examples/rpl-border-router/native/Makefile.native

```diff
@@ -3,5 +3,6 @@ MODULES += os/services/slip-cmd
 MAKE_MAC = MAKE_MAC_OTHER
 MAKE_NET = MAKE_NET_IPV6
 
+PREFIX ?= fd00::1/64
```

Comment:
Just a pedantic remark: a /64 prefix should be fd00::/64, without a trailing 1

Reply:
Right. But the address fd00::1 is also used by tunslip6, right? Then, would you rename the variable `PREFIX`? What is a suitable name?

Reply:
Following the code, PREFIX will become the IP Address of the tun interface (in rpl-native-router and tunslip6) and also be used to create the DODAG Prefix, so both are wrong :)

It could be something like :
TUN_ITF_IP?=fd00::1

Reply:
But the prefix part is also needed no?